Remotion/Web/Development.WebTesting.IntegrationTests.Infrastructure/CustomWebTestConfigurationFactory.cs

```diff
@@ -19,53 +19,73 @@
 using System.IO;
 using System.IO.Compression;
 using Remotion.Web.Development.WebTesting.Configuration;
-using Remotion.Web.Development.WebTesting.WebDriver.Configuration.Chrome;
+using Remotion.Web.Development.WebTesting.WebDriver.Configuration.Chromium;
+using Remotion.Web.Development.WebTesting.WebDriver.Configuration.Chromium.Chrome;
+using Remotion.Web.Development.WebTesting.WebDriver.Configuration.Chromium.Edge;
 
 namespace Remotion.Web.Development.WebTesting.IntegrationTests.Infrastructure
 {
   public class CustomWebTestConfigurationFactory : WebTestConfigurationFactory
   {
-    protected override IChromeConfiguration CreateChromeConfiguration (WebTestConfigurationSection configSettings)
+    protected override IChromiumConfiguration CreateChromeConfiguration (WebTestConfigurationSection configSettings)
     {
       var chromeVersionArchivePath = ConfigurationManager.AppSettings["ChromeVersionArchive"];
-      var advancedChromeOptions = new AdvancedChromeOptions();
+      var advancedChromeOptions = new AdvancedChromiumOptions();
 
       if (string.IsNullOrEmpty (chromeVersionArchivePath))
         return new ChromeConfiguration (configSettings, advancedChromeOptions);
 
-      var customChromeDirectory = PrepareChromeDirectory (chromeVersionArchivePath);
+      var versionedChromeFolder = $"Chrome_v{LatestTestedChromeVersion}";
+      var customChromeDirectory = PrepareCustomBrowserDirectory (chromeVersionArchivePath, versionedChromeFolder);
 
-      var customChromeBinary = GetBinaryPath (customChromeDirectory, "chrome");
+      var customBrowserBinary = GetBinaryPath (customChromeDirectory, "chrome");
       var customDriverBinary = GetBinaryPath (customChromeDirectory, "chromedriver");
-
       var customUserDirectoryPath = CustomUserDirectory.GetCustomUserDirectory();
 
-      var chromeExecutable = new ChromeExecutable (customChromeBinary, customDriverBinary, customUserDirectoryPath);
+      var chromeExecutable = new ChromeExecutable (customBrowserBinary, customDriverBinary, customUserDirectoryPath);
 
       return new ChromeConfiguration (configSettings, chromeExecutable, advancedChromeOptions);
     }
 
-    private string PrepareChromeDirectory (string chromeVersionArchivePath)
+    protected override IChromiumConfiguration CreateEdgeConfiguration (WebTestConfigurationSection configSettings)
     {
-      var versionedChromeFolder = string.Format ("Chrome_v{0}", LatestTestedChromeVersion);
+      var edgeVersionArchivePath = ConfigurationManager.AppSettings["EdgeVersionArchive"];
```

Comment:
We should probably create a new Blogpost for Edge for people to copy this File? Because these changes have to be done manually and stuff. Check with MK